File: source/code/csharp/authentication/get_access_tokens.cs
var allAccessTokens = await account.GetAccessTokens().ToListAsync();
var allRefreshTokens = await account.GetRefreshTokens().ToListAsync();

var accessTokenToDelete = allAccessTokens.Where(at => at.Jwt == access_token);
var refreshTokenToDelete = allRefreshTokens.Where(rt => rt.Jwt == refresh_token);

File: source/code/vbnet/authentication/get_access_tokens.vb
Dim allAccessTokens = Await account.GetAccessTokens().ToListAsync()
Dim allRefreshTokens = Await account.GetRefreshTokens().ToListAsync()

Dim accessTokenToDelete = allAccessTokens.Where(Function(at) at.Jwt = access_token)
Dim refreshTokenToDelete = allRefreshTokens.Where(Function(rt) rt.Jwt = refresh_token)
